src/net.h

```diff
@@ -591,11 +591,11 @@ class CNode
     std::atomic<int64_t> nLastTXTime{0};
 
     /** Last measured round-trip time. Used only for RPC/GUI stats/debugging.*/
-    std::atomic<int64_t> m_last_ping_time{0};
+    std::atomic<std::chrono::microseconds> m_ping_time{0us};
```

Comment:
in the first commit: why is this renamed?

Reply:
I did that to make it more consistent (and as the type changes, all call sites have to be touched anyway).

Reply:
I mean rename from `m_last_ping_time` to `m_ping_time` (removing the "last"), which happened in the latest rebase.

Reply:
Notice - it was renamed _to_ `m_ping_time` in previous incarnations of this PR:

`df8892dc9..7b6775ba7`: `nPingUsecTime` -> `m_ping_time`
`9bbf08bf9..f4450e105`: `m_last_ping_time` -> `m_ping_time`

So, it is all consistent from this PR's point of view. Just `master` changed from `nPingUsecTime` to `m_last_ping_time`. I think either one of `m_last_ping_time` or `m_ping_time` is fine.

Reply:
Yes, I know :)

My opinion is just that it doesn't make sense to rename this after the rebase

Reply:
I renamed this from `nPingUsecTime` to `m_last_ping_time` in #20721. I think `m_last_ping_time` is slightly better than `m_ping_time` because it makes it clear that this is the time from the most recent ping/pong round trip. The next field is `m_min_ping_time`, which is the minimum time of all the ping/pong round trips.

Reply:
Done. Using m_last_ping_time now.

Reply:
Thanks!